api/hackerProfile.js

```diff
@@ -23,13 +23,102 @@ router.get("/", async (req, res) => {
 });
 
 router.put("/", async (req, res) => {
-  // TODO: Do input validation
-  const newHackerProfile = await models.HackerProfile.update(req.body, {
+  // Get the users current hacker profile
+  const currentHackerProfile = await models.HackerProfile.findOne({
+    where: { userId: req.user.id }
+  });
```

Comment:
How are you setting up the frontend, is this gonna always exist?

Reply:
it should -- before they can access the profile form, a `GET /api/profile` request is made which should create a hacker profile if it doesn't exist

Reply:
kk sounds good